src/OSMAreaBuilder.js

```diff
@@ -142,9 +224,59 @@ OSMAreaBuilder.prototype._emitArea = function (area) {
  *
  * @private
  */
-OSMAreaBuilder.prototype._emitDone = function () {
+OSMAreaBuilder.prototype._handleDone = function handleDone() {
+
+  this._handleErrors();
+
   this.emit('done', this._stats);
 };
 
+/**
+ * Set internal stat counts to 0
+ *
+ * @private
+ */
+OSMAreaBuilder.prototype._resetStats = function resetStats() {
+  this._stats = {
+    errorCount: 0,
+    areaCount: 0,
+    timeInArea: 0,
+    timeInAreaHandler: 0,
+    timeInPreprocess: 0
+  };
+
+  this._index = {
+    relations: {},
+    ways: {},
+    areas: []
+  };
+};
+
+/*
+ HELPER FUNCTIONS
+*/
+
+/**
+ * Check if all needed tags are in place
+ *
+ * @param {object} obj
+ * @returns {boolean}
+ */
+function checkAdminTags(obj) {
```

Comment:
problem is it's not always a number, it's sometimes a string. those oddballs get thrown into "other" level file. I don't want to suppress them entirely because the goal is to expose the flaws or inconsistencies of the data as well as its awesomeness.